src/handlers/inGameCommandHandler.js

```diff
@@ -1213,7 +1213,6 @@ module.exports = {
         let instance = client.readInstanceFile(rustplus.guildId);
         let cupboardFound = false;
         for (const [key, value] of Object.entries(instance.serverList[rustplus.serverId].storageMonitors)) {
```

Comment:
You already iterate over storageMonitors about instance.serverList[rustplus.serverId].
value had no property called 'serverId'

Reply:
Ahh yea this is an old relic that got left behind after some refactoring, nice catch!